yarn.lock

```diff
@@ -2049,14 +2067,13 @@ http-signature@~1.2.0:
     jsprim "^1.2.2"
     sshpk "^1.7.0"
 
-https-proxy-agent@^1.0.0:
-  version "1.0.0"
-  resolved "https://registry.yarnpkg.com/https-proxy-agent/-/https-proxy-agent-1.0.0.tgz#35f7da6c48ce4ddbfa264891ac593ee5ff8671e6"
-  integrity sha1-NffabEjOTdv6JkiRrFk+5f+GceY=
+https-proxy-agent@^1.0.0, https-proxy-agent@^2.2.0:
```

Comment:
I think Sean is right, in order to fix this we need to upgrade `gulp-nsp` to a version which doesnt use the affected  package. This line indicates we have both 1.0.0 _and_ 2.2.0 now

Reply:
Looks like NSP has been retired https://github.com/nodesecurity/nsp - I think we should just remove the package and delete its usage from the gulpfile

Reply:
Also we don't really need it anymore given Github now does this for us